packages/features/eventtypes/components/Locations.tsx

```diff
@@ -22,6 +30,24 @@ export type TLocationOptions = Pick<EventTypeSetupProps, "locationOptions">["loc
 export type TDestinationCalendar = { integration: string } | null;
 export type TPrefillLocation = { credentialId?: number; type: string };
 
+type LocationInputCustomClassNames = {
+  addressInput?: string;
+  phoneInput?: string;
+};
+
+export type LocationCustomClassNames = {
+  dropdownWrapper?: string;
+  locationSelect?: locationSelectCustomClassNames;
```

Comment:
Previously you wrote that suffix with `Classnames` is for objects containing classnames - if that's still the case need to rename this to `locationSelectClassnames` and capitalize `locationSelectCustomClassNames` to `LocationSelectCustomClassNames`

Reply:
I see the same is for `organizerContactInput` - it is an object but does not have `Classnames` suffix, but then its child keys have it.

Reply:
but down in other places i don't see `Classnames` suffix e.g. for `type EventSetupTabCustomClassNames`

Reply:
didn't move forward with that approach